packages/hawtio/src/plugins/camel/contexts/contexts-service.ts

```diff
@@ -0,0 +1,99 @@
+import { log } from '../globals'
+import { IRequest, IResponseFn } from 'jolokia.js'
+import { jolokiaService, AttributeValues } from '@hawtiosrc/plugins/connect/jolokia-service'
+import { MBeanNode } from '@hawtiosrc/plugins/shared'
+
+export interface ContextAttributes {
+  Context: string,
+  [key: string]: any
+}
+
+class ContextsService {
```

Comment:
By the way, starting from Camel 3 we no longer allow multiple contexts in a Camel app. The plural treatment of Camel contexts is legacy handling of the code, so we should revisit it and try to remove the presumption that the app can have multiple contexts.

Reply:
What does that mean in terms of backwards compatibility? Does it mean we only support Camel 3+ with the new hawtio?

Reply:
Good point. I understand that backward compatibility is really important from the product point of view, but at the same time I'm not sure if it's good idea to maintain the old design for a new project. Maybe we should keep the design of accepting multiple Camel contexts for the time being?

By the way, we already started to be based on Camel 3.20.2 model for the Camel plugin. To really achieve backward compatibility, we'd need to publish an old Camel 2.x model and cut a branch that consumes the 2.x model for the product.

Reply:
I will try to figure out if we need the backwards compatibility.

Reply:
Tracking issue posted -> https://github.com/hawtio/hawtio-next/issues/126